### PR TITLE
Add Cardano assets

### PR DESCRIPTION
**Pull request by cryptomanz1**

### CODE REVIEW

File: blockchains/algorand/assets/31566704/info.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+    "name": "USD Coin",
+    "website": "https://www.centre.io",
+    "description": "USDC provides a fully collateralized US dollar stablecoin, and is based on the open source asset-backed stablecoin framework developed by Centre.",
+    "explorer": "https://algoexplorer.io/asset/31566704",
+    "research": "https://research.binance.com/en/projects/usd-coin",
+    "type": "ALGORAND",
```
Comment (vcoolish) [inline blockchains/algorand/assets/31566704/info.json:7]: ASA

**Review by vcoolish — APPROVED** [review]

File: blockchains/algorand/assets/31566704/info.json (hunk 1 above)
Comment (defisaur) [reply blockchains/algorand/assets/31566704/info.json]: @vcoolish @George2181 @cryptomanz1 should it be `ASA` [there also? (link)](https://github.com/trustwallet/assets/blob/master/blockchains/algorand/assets/137594422/info.json#L8)

Comment (vcoolish) [reply blockchains/algorand/assets/31566704/info.json]: așa is only for algorand.
stands for algorand standard asset

**Comment by Mooris200** [comment]
در تاریخ جمعه ۱۸ اوت ۲۰۲۳،‏ ۲۳:۵۹ merge-fee-bot[bot] <
***@***.***> نوشت:

**Pull request merged**